docker/pulsar-all/pom.xml

```diff
@@ -139,6 +135,7 @@
                 <goals>
                   <goal>build</goal>
                   <goal>tag</goal>
+                  <goal>push</goal>
```

Comment:
```suggestion
```

Reply:
Only do push on the profile `docker-push`. Commit this change.